frontend/src/app/components/enterprise/enterprise-modal/enterprise-trial-form/ee-trial-form.component.sass

```diff
@@ -0,0 +1,10 @@
+@import 'helpers'
+
+#enterprise-trial-form
+  .-error
+    .form--field-instructions,
+    .form--label,
+    .form--label::after
+      color: var(--content-form-danger-zone-bg-color)
+    .form--text-field
+      border: 1px solid var(--content-form-danger-zone-bg-color)
```

Comment:
We already have a rule for the border, called `-required-highlighting`. So pease use this one. I guess it is sufficient to highlight the border, so this file can then be replaced completely.